File: pythonP/animals/animal.py
import random
class Animal(object):
    def __init__(self, name, age):
        self.age = age
        self.name = name
    def getStatus(self):
        print(self.name+" is fine")
    def whoami(self):
        print("My name is "+self.name+" and I am a "+self.type);
        print("My species is "+self.species);
        print("I belong to the group: "+self.group)
    def eat(self):
        food = random.choice(self.foods)
        print("Your animal ate "+food)

File: javaPrograms/animal/animal.java
public class animal {
    public animal() {
        System.out.println("Animal Class");
    }
    public void getStatus(int code) {
        if (code == 0) {
            System.out.println("Animal is fine");
        }else{
            System.out.println("Animal needs food");
        }
    }
    public static void main(String[] args) {
        System.out.println("Animal Class");
    }
}